ui/common/src/commonMain/kotlin/social/androiddev/common/utils/HtmlParser.kt

```diff
@@ -0,0 +1,77 @@
+package social.androiddev.common.utils
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.withStyle
+import it.skrape.core.htmlDocument
+import it.skrape.selects.eachLink
+import it.skrape.selects.eachText
+import it.skrape.selects.html5.a
+import it.skrape.selects.html5.body
+
+
+/**
+ * Takes a String reciever of html text
+ * converts it to an annotated string of text and links
+ */
+fun String.renderHtml(): AnnotatedString {
```

Comment:
I'm wondering if this should instead be moved to the VM? The TootContent UI model could have AnnotatedString in place of String for the message field.

Reply:
good idea